packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -1903,10 +1903,14 @@ https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scope
         for (let i = 0; i < tokenIds.length; i++) {
           const tokenId = tokenIds[i];
           const pubKey = await this.pkpNftContract.read.getPubkey(tokenId);
-          const addrs = await derivedAddresses({
+          const addrs: TokenInfo = await derivedAddresses({
```

Comment:
Probably should make the `tokenId` optional in `TokenInfo` to keep Typescript consistency